.editorconfig

```diff
@@ -21,3 +21,10 @@ ktlint_standard_enum-wrapping = disabled
 ktlint_standard_statement-wrapping = disabled
 ktlint_standard_try-catch-finally-spacing = disabled
 ktlint_standard_wrapping = disabled
+ktlint_standard_chain-method-continuation = disabled
+ktlint_standard_function-expression-body = disabled
+ktlint_standard_class-signature = disabled
+ktlint_standard_binary-expression-wrapping = disabled
+ktlint_standard_condition-wrapping = disabled
+ktlint_standard_function-literal = disabled
+ktlint_standard_backing-property-naming = disabled
```

Comment:
I feel like `function-literal` and `backing-property-naming` are nice improvements, actually. Could we fix them instead of disabling? Agreed the other feel unneeded.

Reply:
I didn't analyze which of those rules are worth enabling. I was thinking that maybe it would be a good task for me while you are off in September? It would be something easier to review for @lognaturel.